Sources/Views/Cells/ContactMessageCell.swift

```diff
@@ -0,0 +1,150 @@
+/*
+ MIT License
+ 
+ Copyright (c) 2017-2018 MessageKit
+ 
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+ 
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+ 
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+ */
+
+import UIKit
+
+open class ContactMessageCell: MessageContentCell {
+    
+    public enum ConstrainsID: String {
+        case initialsContainerLeftConstraint
+        case disclouserRigtConstraint
+    }
+    
+    /// The view container that holds contact initials
+    open lazy var initialsContainerView: UIView = {
+        let initialsContainer = UIView(frame: CGRect.zero)
+        initialsContainer.backgroundColor = .white
+        initialsContainer.layer.shadowColor = UIColor.gray.cgColor
+        initialsContainer.layer.shadowOffset = CGSize(width: 0, height: 2)
+        initialsContainer.layer.shadowRadius = 2
+        initialsContainer.layer.shadowOpacity = 0.6
```

Comment:
I think we should remove the shadow as none of our other base elements have shadow. Perhaps a `configureInitialsView` function of sorts is needed so this could customized